### ISSUE TITLE
ESP32 - setting wrong WiFi password causes crash

### ISSUE THREAD

**Issue by teuteuguy** [cesanta/mongoose-os#356]
When using the mos wifi command with ssid and password and setting password to something incorrect, causes device to crash and core_dump.

Can’t seem to catch the core_dump cause console isn’t active at that point, cause of the mos wifi command

**Comment by dimonomid** [comment]
Confirmed, thanks for reporting.

**Comment by tssh2005** [comment]
Any news in this issue? Does someone know a workaround?

**Comment by tripflex** [comment]
Yeah this could be a huge deal breakers in moving forward with ESP32 and device provisioning
